sc/social/like/interfaces.py

```diff
@@ -200,6 +201,16 @@ class ISocialLikeSettings(model.Schema):
         default=True,
     )
 
+    facebook_prefetch_enable = schema.Bool(
```

Comment:
I prefer to name this field `facebook_prefetch_enabled`

Reply:
Suggestion for name?